ui/src/app/shared/utils/url-search-params.utils.ts

```diff
@@ -143,7 +143,7 @@ import {TimeFrame} from "../model/TimeFrame";
     let chips: Map<string, string> = new Map();
     urlSearchParams.paramsMap.forEach((values: string[], key: string) => {
       if (values && key) {
-        chips.set(key, values.toString());
+        chips.set(key, decodeURIComponent(values.toString()));
```

Comment:
No chance of a key having spaces? (probably true, but I can't immediately see where that's enforced)

Reply:
Good point – fixed in shim layer, too